libs/coin-modules/coin-evm/src/bridge/synchronization.ts

```diff
@@ -1,3 +1,4 @@
+// TODO: Remove this file once Celo bridge is removed
```

Comment:
on https://ledgerhq.atlassian.net/browse/LIVE-23483 i was planning to remove it with @hedi-edelbloute but found a bug on getting failed transactions. Not sure we have a ticket for the bug and one to follow up this changes once bug is resolved